Source/quests.h

```diff
@@ -58,6 +58,13 @@ struct Quest {
 	uint8_t _qvar2;
 
 	bool IsAvailable();
+
+	void activateMultiStageQuest(uint8_t quest_stage = 1)

```

Comment:
Are there single stage quests, do they need to be activated differently?

Reply:
There seem to be a few different types of quests. Some like Q_ANVIL and Q_ROCK are made active independently of being marked visible in the log, Q_GRAVE uses qvar1 but appears to progress before the quest is made active in some cases. Q_GARBUD, Q_ZHAR, Q_VEIL (and possibly others) don't use qvar1. Q_SKELKING uses qvar1 but isn't marked visible in the log when it's made active.

Reply:
Sorry to break the conversation, I ended up splitting this change into #5085 so this one can be focused on the last couple of non-door OperateObject functions.